lib/libutee/tee_api.c

```diff
@@ -83,6 +83,10 @@ TEE_Result TEE_InvokeTACommand(TEE_TASessionHandle session,
 	res = utee_invoke_ta_command(session, cancellationRequestTimeout,
 				      commandID, paramTypes, params,
 				      returnOrigin);
+	if (returnOrigin != NULL)
```

Comment:
Even if returnOrigin is NULL we must not panic if the error code is from the TA.